worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowManager.java

```diff
@@ -107,6 +110,19 @@ private static File getContextScriptFile(final String contextScriptFilePath) {
                 } catch (final IOException e) {
                     throw new RuntimeException("Could not obtain workflow-control.js");
                 }
+
+                try {
+                    final String addFailures = Resources.toString(Resources.getResource("add-failures.js"),
+                            StandardCharsets.UTF_8);
+                    stringBuilder.append("thisScriptObject = `\n").append(addFailures);
```

Comment:
Does this need some escaping?

Reply:
> 
> 
> Does this need some escaping?

In what sense? It's all wrapped as a multi-line string

Reply:
Oh right.  What about the backtick itself?  We're not concerned that the script will have one?  Or just add that code if/when it happens?

Reply:
Fair point, I've updated it to escape any `

Reply:
I'm looking at this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Can these strings also include expressions?

If so should we also avoid those?  Perhaps using that `raw` function it talks about?  Perhaps this is just overkill since we know exactly what we're inserting and it's not configurable?  I'd be OK with that answer also.